lib/node_modules/@stdlib/constants/float32/sqrt-three/test/test.js

```diff
@@ -0,0 +1,56 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var sqrt = require( '@stdlib/math/base/special/sqrt' );
+var float64ToFloat32 = require( '@stdlib/number/float64/base/to-float32' );
+var absf = require( '@stdlib/math/base/special/absf' );
+var EPS = require( '@stdlib/constants/float32/eps' );
+var FLOAT32_SQRT3 = require( './../lib' );
+
+
+// TESTS //
+
+tape( 'main export is a number', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof FLOAT32_SQRT3, 'number', 'main export is a number' );
+	t.end();
+});
+
+tape( 'the exported value is a single-precision floating-point number equal to the square root of 3', function test( t ) {
+	t.equal( FLOAT32_SQRT3, float64ToFloat32( sqrt( 3 ) ), 'equals sqrt(3)' );
+	t.end();
+});
+
+tape( 'export equals `sqrt(3)`', function test( t ) {
+	var delta;
+	var tol;
+	var v;
+
+	v = sqrt( 3.0 );
+	delta = absf( float64ToFloat32( v - FLOAT32_SQRT3 ) );
+	tol = EPS * FLOAT32_SQRT3;
+
+	t.ok( delta <= tol, 'equals sqrt(3) within tolerance '+tol );
+
+	t.end();
+});
```

Comment:
Since it's the single-precision variant, I am thinking we want to compare the value to the result of [`sqrtf(3)`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/math/base/special/sqrtf)

Reply:
@manvith2003  Why are we checking that the constant is approximately equal to `sqrt(3)`? Shouldn't the constant value equal the expected value exactly?